x-pack/test/scalability/apis/api.telemetry.cluster_stats.no-cache.json

```diff
@@ -0,0 +1,45 @@
+{
+  "journeyName": "POST /api/telemetry/v2/clusters/_stats - no cache",
+  "scalabilitySetup": {
+    "warmup": [
+      {
+        "action": "constantUsersPerSec",
+        "userCount": 1,
+        "duration": "30s"
+      }
+    ],
+    "test": [
+      {
+        "action": "rampUsersPerSec",
+        "minUsersCount": 1,
+        "maxUsersCount": 10,
+        "duration": "120s"
+      }
```

Comment:
Interesting case: what is the load we expect in non-cache scenario? How frequent?

Since these tests are targeting capacity check, it is expected to increase requests count within reasonable time.

Reply:
FWIW, we only request the telemetry payload when we checked that it hasn't been reported in the last 24h.

That means we should expect, maximum, 1 daily peak of requests (where all active users at that moment may request the endpoint).

On top of that, that's true for the _cached_ request. The non-cached one is only used for admins that want to audit what we send about them.